lib/gooddata/models/project.rb

```diff
@@ -860,9 +860,9 @@ def data_permissions(id = :all)
     alias_method :user_filters, :data_permissions
 
     # Deletes project
-    def delete
+    def delete(opts = {})
       fail "Project '#{title}' with id #{uri} is already deleted" if deleted?
-      client.delete(uri)
+      client.delete(uri, opts)
```

Comment:
It's not consistent to introduce option retries only for delete function, but not for other methods. Please move the retry logic into test code only.

Reply:
Because retry logic is handled inside request method of connection.rb so we will use GD_MAX_RETRY in config for this clean-up job to override default retry number (12) instead of using options parameters